packages/databricks-vscode/src/cli/CliWrapper.ts

```diff
@@ -410,15 +413,25 @@ export class CliWrapper {
             bundleOpName: "deploy",
         });
 
+        // Add python executable to PATH
+        const executable = await pythonExtension.getPythonExecutable();
+        let shellPath = process.env.PATH;
+        if (executable) {
+            shellPath = `${path.dirname(executable)}${path.delimiter}${
+                process.env.PATH
```

Comment:
Might be better to use the PATH from the getEnvVarsForCli. Currently it's the same, but if we ever update it that function, this logic will override the changes